components/brave_wallet/browser/simulation_service.cc

```diff
@@ -0,0 +1,326 @@
+             chain_id ==
+                 brave_wallet::mojom::kBinanceSmartChainMainnetChainId) {
+    url = "bnb/v0/mainnet/scan";
+  } else if (coin == brave_wallet::mojom::CoinType::ETH &&
+             chain_id == brave_wallet::mojom::kArbitrumMainnetChainId) {
+    url = "arbitrum/v0/one/scan";
+  }
+
+  return url;
+}
+
+bool HasTransactionScanSupportInternal(const std::string& chain_id,
+                                       brave_wallet::mojom::CoinType coin) {
+  return (coin == brave_wallet::mojom::CoinType::SOL &&
+          chain_id == brave_wallet::mojom::kSolanaMainnet) ||
+         (coin == brave_wallet::mojom::CoinType::SOL &&
+          chain_id == brave_wallet::mojom::kSolanaTestnet) ||
+         (coin == brave_wallet::mojom::CoinType::SOL &&
+          chain_id == brave_wallet::mojom::kSolanaDevnet) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kMainnetChainId) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kGoerliChainId) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kPolygonMainnetChainId) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kBinanceSmartChainMainnetChainId) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kArbitrumMainnetChainId);
+}
+
+bool HasMessageScanSupportInternal(const std::string& chain_id,
+                                   brave_wallet::mojom::CoinType coin) {
+  return (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kMainnetChainId) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kGoerliChainId) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kPolygonMainnetChainId) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kBinanceSmartChainMainnetChainId) ||
+         (coin == brave_wallet::mojom::CoinType::ETH &&
+          chain_id == brave_wallet::mojom::kArbitrumMainnetChainId);
+}
+
+// Function to convert all numbers in JSON string to strings.
+//
+// For sample JSON response, refer to ParseJupiterQuote.
+absl::optional<std::string> ConvertAllNumbersToString(const std::string& json) {
+  auto converted_json = std::string(json::convert_all_numbers_to_string(json));
+  if (converted_json.empty()) {
+    return absl::nullopt;
+  }
+
+  return converted_json;
+}
+
+base::flat_map<std::string, std::string> GetHeaders() {
+  return {
+      {brave_wallet::kBlowfishAPIVersionHeader,
+       brave_wallet::kBlowfishAPIVersion},
+      {net::HttpRequestHeaders::kContentType, "application/json"},
+      {brave_wallet::kBraveServicesKeyHeader, BUILDFLAG(BRAVE_SERVICES_KEY)}};
+}
+
+}  // namespace
```

Comment:
Most of these methods could be in 
```
namespace brave_wallet {
namespace {
...
}
}
```
to get rid of `brave_wallet::` everywhere

Reply:
db90ae36d816b773a42f584af5e88811aa62b59f